### PR TITLE
Added religion.json data

### PR DESCRIPTION
**Pull request by FarmingtonS9**
Added religion data in json format.

Working on requirements:
- POP-286 to POP-299

### CODE REVIEW

File: game/common/religion.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        "sikh": {
+            "colour": [
+                0.3,
+                0.8,
+                0
+            ]
+        }
+    },
+    "animist": {
+        "pagan": {
```
Comment (Spartan322) [inline game/common/religion.json:100]: Shouldn't this be the opposite, all animists are pagans, but not all pagans are animist, and animist is what the game shows as the religion.

**Review by Spartan322 — CHANGES_REQUESTED** [review]

File: game/common/religion.json (hunk 1 above)
Comment (FarmingtonS9) [reply game/common/religion.json:100]: True, fixing it now

**Review by Spartan322 — CHANGES_REQUESTED** [review]
You should remove the submodule push, it is 4 commits behind the master, that's why it won't compile, remember that when submodules change you need to run ` git submodule update --init --recursive` to update your submodule, and don't just stage every change.

**Comment by Spartan322** [comment]
Also when you're ready, you should squash this, an easy way to do this is `git rebase -i HEAD~2` 

The 2 here can be replaced with however many (non-merge) commits you have and replace every `pick` but the first one with `s` or `squash`.

**Comment by FarmingtonS9** [comment]
Yea, still new to Git commands. I hadn't realised that I hadn't updated the submodules.

**Comment by Hop311** [comment]
Closing as we won't be implementing custom mode for a while.

**Pull request closed without merging**